src/Component/Molecule/IconTextSet.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+import SquareButton from '../Atom/SquareButton';
+
+const IconTextSet = ( ) => {
+
+    const onUserClick = ( e : React.FormEvent ): void => {
+        
+    }
+    const onMessageClick = ( e : React.FormEvent ): void => {
+        
+    }
+    const onSettingClick = ( e : React.FormEvent ): void => {
```

Comment:
이벤트 핸들러의 이름은 `handle + 대상 + 동작` 형태로 짓습니다.
```suggestion
    const handleUserClick = ( e : React.FormEvent ): void => {
        
    }
    const handleMessageClick = ( e : React.FormEvent ): void => {
        
    }
    const handleSettingClick = ( e : React.FormEvent ): void => {
```